### ISSUE TITLE
NC 25 Could not find resource file "/apps/logreader/css/../js/logreader-main.css"

### ISSUE THREAD

**Issue by AndyXheli** [nextcloud/logreader#780]
{"reqId":"0yZZ95jPZFoillu2eMb9","level":0,"time":"2022-09-13T14:45:31+00:00","remoteAddr":"10.99.0.65","user":"admin","app":"logreader","method":"GET","url":"/settings/admin/logging","message":"Could not find resource file \"/apps/logreader/css/../js/logreader-main.css\"","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"25.0.0.11","data":{"app":"logreader"},"id":"6320978c44dd6"}

**Comment by fkbreitl** [comment]
Same here

**Comment by fr3ddy-sudo** [comment]
same here

**Comment by HsunGong** [comment]
Tag it

**Comment by crowdstar5566** [comment]
Same

**Comment by chrissi5120** [comment]
same

**Comment by BuyMyMojo** [comment]
another bump

**Comment by DanielSBentz** [comment]
And another Bump

**Comment by BuyMyMojo** [comment]
A fix I found was downloading the latest .zip version of nextcloud and just replacing my apps folder with the one from the zip. a bit hackey but suggests something might not update properly?

**Comment by DanielSBentz** [comment]
Maybe Upgrading to 26RC1 might also be an Idea.

@BuyMyMojo What Version did you use ?

**Comment by BuyMyMojo** [comment]
Just the latest one from the website right now, 25.0.3 I think?Installnextcloud.comUnder community projects and the “Archive” section zip fileOn 17 Feb 2023, at 23:43, DanielSBentz ***@***.***> wrote:﻿
Maybe Upgrading to 26RC1 might also be an Idea.
@BuyMyMojo What Version did you use ?

**Comment by drhirn** [comment]
Same with 26.0.0.
`/apps/logreader/css/../js/logreader-main.css` is a strange path, isn't it?

**Comment by bastiencyr** [comment]
Hello, 
Creating an empty directory takes away this error: `mkdir apps/logreader/css`

**Comment by marianionut** [comment]
Same with Nextcloud Hub 4 => version 26.0.2 ( snap installation )

```
<html>
<body>
<!--StartFragment-->

Debug | logreader | Could not find resource file "/apps/logreader/css/../js/logreader-main.css" |   | 2023-07-03T14:11:11+0300
-- | -- | -- | -- | --


<!--EndFragment-->
</body>
</html>
```

**Comment by ostasevych** [comment]
The same issue at 27.1.2 with the recent update.

**Comment by susnux** [comment]
Fixed with #1042 and #1043
